tox.ini

```diff
@@ -4,7 +4,7 @@ package_name = invirtualenv
 
 [tox]
 skip_missing_interpreters = True
-envlist = pycodestyle, pylint, py27, py35, py36, py37
+envlist = py27, py35, py36, py37, py38
```

Comment:
pycodestyle and pylint sections are still defined below even though they are remove from envlist.